woocommerce/woocommerce-admin
# Update marketing tab styles

In progress work for updating styles used on the marketing tab and coupons pages to reflect designs below

### Changes

* Updated styles of the coupons page recommended extensions (see below)
* Moved to use local copies of product icons/svgs so that inline svgs could be used far more easily - making use of a few existing icons from @wordpress/icons
* Fixed instances of "woocommere-admin"
* Moved to use theme colors

### Screenshots

![Screen Recording 2020-06-23 at 12 39 PM](https://user-images.githubusercontent.com/355014/87127689-78439280-c2cd-11ea-8946-2d242a1756c9.gif)

![redline-for-p2](https://user-images.githubusercontent.com/355014/87127713-82fe2780-c2cd-11ea-89d8-ce0cdd6db28e.png)

### Other

* Tested in IE11
* Using SVG component with recommended accessibility fields

### Detailed test instructions:

- Check out this branch
- Open test site
- Go to Marketing > Overview and Marketing > Coupons
- Check design

### Changelog Note:

Tweak: Style improvements for Marketing hub.
